### PR TITLE
Update `persist init` command with full functionality

### PR DESCRIPTION
**Pull request by sahanHe**
## Purpose
$subject 
Related to [3155](https://github.com/ballerina-platform/ballerina-standard-library/issues/3155)

### CODE REVIEW

File: native/src/main/java/io/ballerina/persist/cmd/Init.java [hunk 1]
```diff
@@ -53,14 +50,15 @@ public class Init implements BLauncherCmd {
     private final PrintStream outStream = System.out;
     private final PrintStream errStream = System.err;
 
+    private static Path configPath = Paths.get("config.toml");
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/cmd/Init.java:53]: It needs to be changed as `Config.toml` and made a constant.

File: native/src/main/java/io/ballerina/persist/cmd/Init.java [hunk 2]
```diff
@@ -73,6 +71,12 @@ public void execute() {
             } catch (Exception e) {
                 errStream.println(e.getMessage());
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/cmd/Init.java:72]:
```suggestion
                errStream.println("Failure when creating the Config.toml file: " + e.getMessage());
```

File: native/src/main/java/io/ballerina/persist/cmd/Init.java [hunk 3]
```diff
@@ -73,6 +71,12 @@ public void execute() {
...
             }
+        } else {
+            try {
+                updateConfigToml();
+            } catch (Exception e) {
+                outStream.println("File reading Error");
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/cmd/Init.java:78]:
```suggestion
                errStream.println("Failure when updating the Config.toml file: " + e.getMessage());
```

File: native/src/main/java/io/ballerina/persist/cmd/Init.java [hunk 4]
```diff
@@ -81,6 +85,11 @@ private void createConfigToml() throws Exception {
         writeOutputFile(toml, Paths.get("").toAbsolutePath().toString() + "/config.toml");
     }
 
+    private void updateConfigToml() throws Exception {
+        SyntaxTree toml = CreateSyntaxTree.updateToml(configPath);
+        writeOutputFile(toml, Paths.get("").toAbsolutePath().toString() + "/config.toml");
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/cmd/Init.java:90]:
```suggestion
        writeOutputFile(toml, Paths.get("").toAbsolutePath().toString() + "/Config.toml");
```

Comment (MadhukaHarith92) [reply native/src/main/java/io/ballerina/persist/cmd/Init.java:90]: Use constant value ("Config.toml") here.

File: native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java [hunk 5]
```diff
@@ -39,24 +46,83 @@
 import static io.ballerina.persist.PersistToolsConstants.DEFAULT_USER;
 
 /**
- * Class to create syntax tree for ballerina.
+ * Class to create syntax tree for config.toml.
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java:49]:
```suggestion
 * Class to create syntax tree for Config.toml.
```

File: native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java [hunk 6]
```diff
@@ -39,24 +46,83 @@
...
  */
 public class CreateSyntaxTree {
     private static final PrintStream outStream = System.out;
+
+    /**
+     * Method to create new config.toml.
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java:55]:
```suggestion
     * Method to create new Config.toml.
```

File: native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java [hunk 7]
```diff
@@ -39,24 +46,83 @@
...
+        moduleMembers = populateNodeList(moduleMembers, true);
+        Token eofToken = AbstractNodeFactory.createIdentifierToken("");
+        DocumentNode documentNode = NodeFactory.createDocumentNode(moduleMembers, eofToken);
+        TextDocument textDocument = TextDocuments.from(documentNode.toSourceCode());
+        SyntaxTree syntaxTree = SyntaxTree.from(textDocument);
+        return syntaxTree;
+    }
 
+    /**
+     * Method to update config.toml.
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java:68]:
```suggestion
     * Method to update the Config.toml.
```

File: native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java [hunk 8]
```diff
@@ -39,24 +46,83 @@
...
+     */
+    public static SyntaxTree updateToml(Path configPath) throws IOException {
+        boolean withinTable = false;
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java:71]: Shall we change the variable to something like `isTableEntry`?

File: native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java [hunk 9]
```diff
@@ -39,24 +46,83 @@
...
         NodeList<DocumentMemberDeclarationNode> moduleMembers = AbstractNodeFactory.createEmptyNodeList();
+        moduleMembers = populateNodeList(moduleMembers, false);
+        Path fileNamePath = configPath.getFileName();
+        TextDocument configDocument = TextDocuments.from(Files.readString(configPath));
+        SyntaxTree syntaxTree = SyntaxTree.from(configDocument, fileNamePath.toString());
+        DocumentNode rootNote = (DocumentNode) syntaxTree.rootNode();
+        NodeList nodeList = rootNote.members();
+
+        for (Object i : nodeList) {
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java:80]: Let's change the variable `i` -> `member`

File: native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java [hunk 10]
```diff
@@ -39,24 +46,83 @@
...
+            }
+        }
+        Token eofToken = AbstractNodeFactory.createIdentifierToken("");
+        DocumentNode documentNode = NodeFactory.createDocumentNode(moduleMembers, eofToken);
+        TextDocument textDocument = TextDocuments.from(documentNode.toSourceCode());
+        SyntaxTree syntaxTreeFinal = SyntaxTree.from(textDocument);
+        return syntaxTreeFinal;
+    }
+
+    private static boolean checkKey(KeyNode key) {
```
Comment (MadhukaHarith92) [inline native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java:102]: Shall we change the function name to `isDatabaseConfigurationEntry`?

File: native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java (hunk 8 above)
Comment (sahanHe) [reply native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java]: updated in [dbd5790](https://github.com/ballerina-platform/persist-tools/pull/5/commits/dbd57908d8eee997f82b75e59978f916f25b75f2)

File: native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java (hunk 9 above)
Comment (sahanHe) [reply native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java]: updated in [dbd5790](https://github.com/ballerina-platform/persist-tools/pull/5/commits/dbd57908d8eee997f82b75e59978f916f25b75f2)

File: native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java (hunk 10 above)
Comment (sahanHe) [reply native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java]: updated in [dbd5790](https://github.com/ballerina-platform/persist-tools/pull/5/commits/dbd57908d8eee997f82b75e59978f916f25b75f2)

File: native/src/main/java/io/ballerina/persist/cmd/Init.java (hunk 1 above)
Comment (sahanHe) [reply native/src/main/java/io/ballerina/persist/cmd/Init.java]: updated in [dbd5790](https://github.com/ballerina-platform/persist-tools/pull/5/commits/dbd57908d8eee997f82b75e59978f916f25b75f2)

**Review by MadhukaHarith92 — APPROVED** [review]

**Pull request merged**